.travis.yml

```diff
@@ -3,7 +3,7 @@ sudo: required
 rust:
 - nightly
 - beta
-- 1.0.0
+- stable
```

Comment:
Do we want to always require the latest stable Rust, or do we want to support back to some specific version? I could be convinced to go either way.